docker/lr-utils/Dockerfile

```diff
@@ -13,6 +13,63 @@ RUN echo "source activate lr-utils" > ~/.bashrc
 ENV PATH=/opt/conda/envs/lr-utils/bin/:/root/google-cloud-sdk/bin/:${PATH}
 
 # install gsutil
-RUN apt update
+RUN apt-get --allow-releaseinfo-change update
 RUN apt install -y curl git-lfs parallel
 RUN curl https://sdk.cloud.google.com | bash
+
+# Setup crcmodc for gsutil:
+RUN apt-get install -y gcc python3-dev python3-setuptools && \
+			pip3 uninstall -y crcmod && \
+			pip3 install --no-cache-dir -U crcmod
+
+# Install bamtools:
+# First get cmake:
+RUN wget https://github.com/Kitware/CMake/releases/download/v3.20.1/cmake-3.20.1-linux-x86_64.tar.gz \
+		&& tar -zxf cmake-3.20.1-linux-x86_64.tar.gz \
+		&& mv cmake-3.20.1-linux-x86_64/bin/* /usr/local/bin/ \
+		&& mv cmake-3.20.1-linux-x86_64/doc/* /usr/local/doc/ \
+		&& mv cmake-3.20.1-linux-x86_64/man/* /usr/local/man/ \
+		&& mv cmake-3.20.1-linux-x86_64/share/* /usr/local/share/ \
+		&& rm -rf cmake-3.20.1-linux-x86_64 cmake-3.20.1-linux-x86_64.tar.gz
+
+# Get g++
+RUN apt-get install -y g++
```

Comment:
This leads to docker size bloat, let's build in as few layers as possible.
Consider this request 2ndary though.

Reply:
I'll condense the install lines.